src/api/covariates/__init__.py

```diff
@@ -36,29 +17,75 @@ def update_site_covariates(site):
 
     data_date = result["date"]
     requested_date = result["requested_date"]
+
     aca_covariates = result.get("covariates") or dict()
     aca_benthic = aca_covariates.get("aca_benthic") or []
     aca_geomorphic = aca_covariates.get("aca_geomorphic") or []
 
     aca_benthic_covariate = Covariate.objects.get_or_none(
         name="aca_benthic", site_id=site_pk
-    ) or Covariate(name="aca_benthic", site=site)
+    ) or Covariate(name="aca_benthic", site_id=site_pk)
     aca_benthic_covariate.display = "Alan Coral Atlas Benthic"
     aca_benthic_covariate.datestamp = data_date
     aca_benthic_covariate.requested_datestamp = requested_date
     aca_benthic_covariate.value = aca_benthic
     aca_benthic_covariate.save()
 
     aca_geomorphic_covariate = Covariate.objects.get_or_none(
-        name="aca_geomorphic", site=site
-    ) or Covariate(name="aca_geomorphic", site=site)
+        name="aca_geomorphic", site_id=site_pk
+    ) or Covariate(name="aca_geomorphic", site_id=site_pk)
     aca_geomorphic_covariate.display = "Alan Coral Atlas Geomorphic"
     aca_geomorphic_covariate.datestamp = data_date
     aca_geomorphic_covariate.requested_datestamp = requested_date
     aca_geomorphic_covariate.value = aca_geomorphic
     aca_geomorphic_covariate.save()
 
 
+def _update_site_vot_covariates(point, site_pk):
+    vibrant_oceans_threats = VibrantOceansThreatsCovariate()
+    results = vibrant_oceans_threats.fetch([(point.x, point.y)])
+
+    if not results or not results[0]:
+        return
+
+    result = results[0]
+
+    data_date = result["date"]
+    requested_date = result["requested_date"]
```

Comment:
Here and in the aca version, if we get a result back but it has null datestamps, should we bail?